Tasks/PythonScriptV0/pythonscript.ts

```diff
@@ -0,0 +1,69 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import * as task from 'vsts-task-lib/task';
+import * as toolRunner from 'vsts-task-lib/toolrunner';
+
+import * as uuidV4 from 'uuid/v4';
+
+interface TaskParameters {
+    targetType: string,
+    filePath?: string,
+    script?: string,
+    arguments?: string,
+    pythonInterpreter?: string,
+    workingDirectory?: string,
+    failOnStderr?: boolean
+}
+
+export async function pythonScript(parameters: Readonly<TaskParameters>): Promise<void> {
+    // Get the script to run
+    const scriptPath = await (async () => {
+        if (parameters.targetType.toLowerCase() === 'filepath') { // Run script file
+            // Required if `targetType` is 'filepath':
+            const filePath = parameters.filePath!;
```

Comment:
As I understand it, the [trailing bang](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator) doesn't actually [perform a null check](https://stackoverflow.com/a/42274019).  I think it would be an improvement to have an error message that states '`filePath` is required'.

Reply:
Right -- it's just a compile-time assertion.  I can change it to a runtime check.